esp/src/src-react/theme-shims/themeDuplicates.ts

```diff
@@ -0,0 +1,1073 @@
+/**
```

Comment:
Glad you added this comment, I was just about to ask if these were different somehow from those shims that were in Fluent.